src/components/dashboard/Amount.js

```diff
@@ -39,7 +42,8 @@ const Amount = (props: AmountProps) => {
       setError(`Sorry, you don't have enough G$ to send ${weiToGd(amountWithFee)} (${txFeePercents}% transaction fee)`)
       return false
     } catch (e) {
-      setError(e.message)
+      log.error('Failed canContiniue', e.message, e)
+      setError(`Sorry, Something unexpected happened, please try again.`)
```

Comment:
@StanislavShevchenko please change this error message to: "The error happens in blockchain request, please try again later"